charts/cmc-citizen-frontend/values.preview.template.yaml

```diff
@@ -1,5 +1,5 @@
 tags:
-  ccd-idam-pr: true
```

Comment:
I think we need to remove this completely... it's not needed anymore. It's already enabled in the chart we are using.
https://github.com/hmcts/chart-ccd/blob/master/ccd/values.yaml#L2

Reply:
removed